rpc_jobs/rpc_aio.yml

```diff
@@ -61,19 +76,37 @@
           branches: "do_not_build_on_pr"
           NUM_TO_KEEP: 10
     exclude:
+      - series: kilo
+        context: majorupgrade
       - series: liberty
-        context: upgrade
+        context: majorupgrade
       - series: newton140
-        context: upgrade
+        context: majorupgrade
       - series: newton141
-        context: upgrade
+        context: majorupgrade
       - series: master
-        context: upgrade
+        context: majorupgrade
       # Xenial builds are run for newton and above.
+      - series: kilo
+        context: xenial
       - series: liberty
         context: xenial
       - series: mitaka
         context: xenial
+      # Leap upgrades are only run for newton141
+      - series: kilo
+        context: leapupgrade
+      - series: liberty
+        context: leapupgrade
+      - series: mitaka
+        context: leapupgrade
+      - series: newton140
+        context: leapupgrade
+      - series: master
+        context: leapupgrade
+      # Ceph builds are not run for kilo
```

Comment:
This comment doesn't add anything. Why aren't they? I believe there is Ceph support in kilo.